mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -230,7 +235,8 @@ def save_to_firebase(self, project, groups, groupsOfTasks):
                 # since the tasks hold geometries their storage size
                 # can get quite big otherwise
                 if self.projectType in [ProjectType.FOOTPRINT.value]:
-                    # todo: remove extra information from geojson
+                    # removing properties from each task
+                    tasks_list = [task.pop("properties", None) for task in tasks_list]
```

Comment:
I'm not sure if this works as expected. What is written into the list? would it be enough just to have the loop and not write the output back to the list?

Reply:
yup, thats a big no no